docs/private-node-attributes.md

```diff
@@ -0,0 +1,26 @@
+# Private attributes of the NeoFS storage nodes
+
+Storage nodes declare key-value string attributes when applying to enter the
+network map. In general, any attributes can be declared, however, some of them
+may be subject to restrictions. In particular, to specify so-called private
+attributes, the node must be in the corresponding access list.
+
+## Access lists
+
+These lists are stored in the NeoFS NNS. For each private attribute value
+`KEY=VALUE`, there is a domain with name `MD5(VALUE).MD5(KEY).private-node-attributes.neofs`
+and records of `TXT` type with HEX-encoded public keys. Here `MD5` is a
+HEX-encoded MD5 checksum. If the domain exists and has non-empty list of public
```

Comment:
How about `base58(SHA256(Key + \x0 + Value))`? Zero byte can be replaced by any other separator that can't be used in keys or values.

Reply:
about concatenation: i thought about that. Right now it may be unstable since attribute value may be [any](https://github.com/nspcc-dev/neofs-api/blob/c1a97ad6ea03f8bd7bf71e47cba946a9cb1ca9e5/netmap/types.proto#L140-L141), so we cannot select a reserved byte sequence (like `\0` in ur example). But if it existed, we could save on the number of domains. In total, i see either two domains or protocol changes

about encoding: i don't see much differencies b/w md5/sha256 and hex/base58 right now. What potential problem do they solve or why else u suggest to use them?